apps/src/redux/instructions.js

```diff
@@ -4,73 +4,63 @@
  * that are required for this feature, and the reducer that sets state based
  * off of those actions.
  */
-var _ = require('../lodash');
+const TOGGLE_INSTRUCTIONS_COLLAPSED = 'instructions/TOGGLE_INSTRUCTIONS_COLLAPSED';
+const SET_INSTRUCTIONS_HEIGHT = 'instructions/SET_INSTRUCTIONS_HEIGHT';
+const SET_INSTRUCTIONS_MAX_HEIGHT = 'instructions/SET_INSTRUCTIONS_MAX_HEIGHT';
 
-var TOGGLE_INSTRUCTIONS_COLLAPSED = 'instructions/TOGGLE_INSTRUCTIONS_COLLAPSED';
-var SET_INSTRUCTIONS_HEIGHT = 'instructions/SET_INSTRUCTIONS_HEIGHT';
-var SET_INSTRUCTIONS_MAX_HEIGHT = 'instructions/SET_INSTRUCTIONS_MAX_HEIGHT';
-
-var instructionsInitialState = {
+const instructionsInitialState = {
   collapsed: false,
   // represents the uncollapsed height
   height: 300,
   maxHeight: 0
 };
 
-module.exports.default = function reducer(state, action) {
-  state = state || instructionsInitialState;
-
+export default function reducer(state = instructionsInitialState, action) {
   if (action.type === TOGGLE_INSTRUCTIONS_COLLAPSED) {
-    return _.assign({}, state, {
+    return Object.assign({}, state, {
       collapsed: !state.collapsed
     });
   }
 
   if (action.type === SET_INSTRUCTIONS_HEIGHT &&
       action.height !== state.height) {
-    return _.assign({}, state, {
+    return Object.assign({}, state, {
       height: action.height
     });
   }
 
   if (action.type === SET_INSTRUCTIONS_MAX_HEIGHT &&
       action.maxHeight !== state.maxHeight) {
-    return _.assign({}, state, {
+    return Object.assign({}, state, {
       maxHeight: action.maxHeight
     });
   }
 
   return state;
-};
+}
 
 /**
  * Toggles whether instructions are currently collapsed.
  */
-module.exports.toggleInstructionsCollapsed = function () {
-  return {
-    type: TOGGLE_INSTRUCTIONS_COLLAPSED
-  };
-};
+export const toggleInstructionsCollapsed = () => ({
+  type: TOGGLE_INSTRUCTIONS_COLLAPSED
+});
 
 /**
  * Set the height of the instructions panel
  * @param {number} height - Height of instructions pane
  */
-module.exports.setInstructionsHeight = function (height) {
-  return {
-    type: SET_INSTRUCTIONS_HEIGHT,
-    height: height
-  };
-};
+export const setInstructionsHeight = height => ({
+  type: SET_INSTRUCTIONS_HEIGHT,
+  height
+});
 
 /**
  * Set the max height of the instructions panel
  * @param {number} maxHeight - Don't let user drag instructions pane to be
  *   larger than this number.
  */
-module.exports.setInstructionsMaxHeight = function (maxHeight) {
-  return {
-    type: SET_INSTRUCTIONS_MAX_HEIGHT,
-    maxHeight: maxHeight
-  };
-};
+export const setInstructionsMaxHeight = maxHeight => ({
```

Comment:
If we want to follow AirBnB, this should not be a function expression.  [Rule 7.1](https://github.com/airbnb/javascript#functions--declarations) and the [section on modules](https://github.com/airbnb/javascript#modules) seem to suggest that regular function declarations are preferred at the module level, e.g.:

```javascript
export function setInstructionsMaxHeight(maxHeight) {
  return {...};
}
```

See [this example in rule 22.6](https://github.com/airbnb/javascript#naming--filename-matches-export) for a case where they do this.

What's the benefit of making this function expression `const`?